protocol/messenger.go

```diff
@@ -3059,12 +3059,26 @@ func (m *Messenger) handleRetrievedMessages(chatWithMessages map[transport.Filte
 
 	logger := m.logger.With(zap.String("site", "RetrieveAll"))
 
+	adminCommunitiesChatIDs, e := m.communitiesManager.GetAdminCommunitiesChatIDs()
+	if e != nil {
+		logger.Info("failed to retrieve admin communities", zap.Error(e))
+	}
```

Comment:
This is another preference thing, although it is also strong go convention. `error` vars are declared `err`, which is a little bit of an exception to the other convention of using acronyms and very short variable names.

If it is human readable and is meaningful that's all that counts, the compiler doesn't care what you call your vars.

```suggestion
	adminCommunitiesChatIDs, err := m.communitiesManager.GetAdminCommunitiesChatIDs()
	if err != nil {
		logger.Info("failed to retrieve admin communities", zap.Error(e))
	}
```

Reply:
Updated